engine/execution/computation/computer/computer.go

```diff
@@ -175,17 +179,21 @@ func (e *blockComputer) executeBlock(
 				panic(err)
 			}
 			res.EventsHashes = append(res.EventsHashes, hash)
+			blockLogger.Debug().Msg("event hashed (callback)")
```

Comment:
Would itbbe worth adding the collection ID to the callback and logging it so it's easier to correlate the two debug lines before and after hashing? Either that or the index